### DISCUSSION TITLE
How does the tool work? How to incorporate in DevOps lifecycle?

### DISCUSSION THREAD

**Discussion by siggipop** [CorterraSolutions/SolutionDependencyChecker#14] [General]
label: question 
How does this tool work? Does it use standard PAC CLI method or does it query components directly with OragnisationService? 
How can this be implemented in a DevOps cycle?

**Comment by bonomo1** [comment]
Uses the RetrieveMissingComponentsRequest

https://learn.microsoft.com/en-us/dotnet/api/microsoft.crm.sdk.messages.retrievemissingcomponentsrequest?view=dataverse-sdk-latest

**Comment by bonomo1** [comment]
In regards to incorporating into a DevOps cycle, we currently use this tool in a manual mode before we build. To automate in your favorite CI/CD tool you could write a PowerShell script to check the dependencies using the above request.
